plugins/org.eclipse.che.vscode-redhat.java/0.38.0/meta.yaml

```diff
@@ -4,7 +4,7 @@ type: VS Code extension
 name: Language Support for Java(TM)
 title: Language Support for Java(TM) by Red Hat
 description: Java Linting, Intellisense, formatting, refactoring, Maven/Gradle support and more...
-url: https://marketplace.visualstudio.com/_apis/public/gallery/publishers/redhat/vsextensions/java/0.38.0/vspackage
+url: https://github.com/redhat-developer/vscode-java/releases/download/v0.38.0/redhat.java-0.38.0.vsix
 icon: https://www.eclipse.org/che/images/logo-eclipseche.svg
```

Comment:
Hmh...I'm not sure we should "fix" old versions?

Reply:
why not? this has been done for all the other plugins

Reply:
Because generally, we don't fix old versions of our software unless we have a specific need to do so. If the user wants a fix, he updates to a later version of the software (in this case the Che plugin)

Reply:
hmm.. but it is basically just an update of download url, binaries are the same e.g. no API breakage.
So, if a user does not want to update for some reason he would still use the very same version of the extension, but downloaded from the different source and will not hit rame limit error during the extention processing on Che end.